pkg/metastore/badger.go

```diff
@@ -31,6 +31,7 @@ import (
 // store.
 type BadgerMetastore struct {
 	tracer trace.Tracer
+	logger *BadgerLogger
```

Comment:
The `BadgerLogger` is just used to comply to Badger's logging interface, we use go-kit logger everywhere.